README.md

```diff
@@ -1,24 +1,88 @@
-# README
+# Rails project: Budget App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> In this project, I created a budget calculator website, which has categories of budgets in which you can enter transaction cost for each sub-group in other to get the total budget price in that particular budget category.
 
-Things you may want to cover:
+## Link to video
 
-* Ruby version
+- [Explain](https://www.loom.com/share/c0e9967ce7c84c6ca09f8257ad4f1dc6)
```

Comment:
- Please record a new video, presenting the features of the project. Also, highlight a piece of the code you find interesting or are proud of making as you did in the previous video